dss_datamover/master_application.py

```diff
@@ -84,7 +84,15 @@ def __init__(self, operation, config):
         self.client_password = config["client"]["password"]
 
         self.s3_config = config.get("s3_storage", {})
+
         self.fs_config = self.config.get("fs_config", {})
+        # override NFS configs with CLI args if applicable
+        nfs_cli_args = ['nfs_server', 'nfs_port', 'nfs_share']
+        if set(nfs_cli_args).issubset(self.config):
```

Comment:
This line and the above one is not needed. We are blinding overwriting if the fields are given or add the nfs shares fields if not present in config.